packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -211,9 +215,11 @@ export const registerScaledPriceAuthority = async (
     }),
   );
 
+  const label = `scaledPriceAuthority-${issuerName}`;
```

Comment:
`scaledPriceFeedName` is nicely factored out but... oops! not used here.

though not a correctness issue, **important** for maintenance / documentation